src/lib/config.ts

```diff
@@ -66,6 +66,27 @@ class default_config {
     priority = "0"
 
     // Note to developers: When creating new <modifier-letter> maps, make sure to make the modifier uppercase (e.g. <C-a> instead of <c-a>) otherwise some commands might not be able to find them (e.g. `bind <c-a>`)
+    
+    /**
+     * cmdmaps contains all of the bindings for the command line.
+     */
+    exmaps = {
+        "<Enter>": "ex_accept_line",
+        "<C-j>": "ex_accept_line",
+        "<C-m>": "ex_accept_line",
+        "<Escape>": "ex_hide_and_clear",
+        "<ArrowUp>": "ex_previous_history",
+        "<ArrowDown>": "ex_next_history",
```

Comment:
eventually we should probably separate literal "previous history" versus the "history search" we have currently